tuna/machine_management_interface.py

```diff
@@ -217,37 +253,36 @@ def run_ipmi_command(self, command):
                 GATEWAY_IP, GATEWAY_PORT, GATEWAY_USER)
 
           ssh = MachineManagementInterface.gateway_session
-          _, out_ch, err_ch = ssh.exec_command(cmd, timeout=SSH_TIMEOUT)
-          error = [x.strip() for x in err_ch.readlines()]
-          error = '\n'.join(error)
-          exit_status = out_ch.channel.exit_status
+          if ssh is not None:
+            _, o_var, e_var = ssh.exec_command(cmd, timeout=SSH_TIMEOUT)
```

Comment:
why rename out_ch and err_ch, they are more readable that way compared to o_var and e_varl. Please revert renaming.

Reply:
This is also, for the same reason. same name is used in multiple places , with different function calls. To address the same, variable renaming is needed